nerfstudio/scripts/downloads/download_data.py

```diff
@@ -143,10 +144,10 @@ def download_capture_name(save_dir: Path, dataset_name: str, capture_name: str,
     file_id_or_zip_url = capture_name_to_file_id[capture_name]
     if file_id_or_zip_url.endswith(".zip"):
         url = file_id_or_zip_url  # zip url
-        target_path = str(save_dir / f"{dataset_name}/{capture_name}")
+        target_path = str(save_dir / f"{dataset_name}" / f"{capture_name}")
```

Comment:
Consider simplifying this and other similar changes:
```suggestion
        target_path = str(save_dir / dataset_name / capture_name)
```

Reply:
Oops, midnight brain wasn't thinking too hard. I'll fix this.